### PR TITLE
EOS-15622: Changes required for setup with 2 iface

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

**Comment by rahuljyoti65** [comment]
@sradhanand-pati  I guess you will need to check the condition before this as well https://github.com/Seagate/cortx-prvsnr/blob/fc382668df7b938f5b35250dc4e17a25e0db7200/srv/components/ha/haproxy/files/haproxy.cfg#L115

File: srv/components/system/firewall/teardown.sls [hunk 1]
```diff
@@ -53,7 +53,9 @@ Remove public and private data interfaces:
   cmd.run:
     - names:
       - firewall-cmd --remove-interface={{ pillar['cluster'][grains['id']]['network']['data_nw']['iface'][0] }} --zone=public-data-zone --permanent
+{%- if pillar['cluster'][node]['network']['data_nw']['iface']|length > 1 %}
```
Comment (rahuljyoti65) [inline srv/components/system/firewall/teardown.sls:56]: variable node is undefined in this file. Please use grains['id'] or define node variable first.

File: srv/components/ha/haproxy/files/haproxy.cfg [hunk 2]
```diff
@@ -81,8 +81,12 @@ defaults
 {%- if pillar['cluster'][grains['id']]['network']['data_nw']['pvt_ip_addr'] %}    
 {% set pvt_ip = pillar['cluster'][grains['id']]['network']['data_nw']['pvt_ip_addr'] %}
 {%- else %}
+{%- if pillar['cluster'][node]['network']['data_nw']['iface']|length > 1 %}
```
Comment (rahuljyoti65) [inline srv/components/ha/haproxy/files/haproxy.cfg:84]: variable node is undefined in this file. Please use grains['id'] or define node variable first.

File: srv/components/misc_pkgs/lustre/config.sls [hunk 3]
```diff
@@ -24,7 +24,11 @@ Update lnet config file:
 {%- if 'data0' in grains['ip4_interfaces'] and grains['ip4_interfaces']['data0'] %}
   {%- set data_if = 'data0' -%}
 {% else %}
+  {%- if pillar['cluster'][node]['network']['data_nw']['iface']|length > 1 %}
```
Comment (rahuljyoti65) [inline srv/components/misc_pkgs/lustre/config.sls:27]: variable node is undefined in this file. Please use grains['id'] or define node variable before.

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by ypise** [comment]
The design and code has moved far past this point. It's difficult to rebase this branch.

**Pull request closed without merging**